dex/testing/btc/base-harness.sh

```diff
@@ -114,7 +114,7 @@ tmux send-keys -t $SESSION:0 "${DAEMON} -rpcuser=user -rpcpassword=pass \
   -rpcport=${ALPHA_RPC_PORT} -datadir=${ALPHA_DIR} $(deprecateddbd $ALPHA_DESCRIPTOR_WALLET) \
   -debug=rpc -debug=net -debug=mempool -debug=walletdb -debug=addrman -debug=mempoolrej \
   -whitelist=127.0.0.0/8 -whitelist=::1 \
-  -txindex=1 -regtest=1 -port=${ALPHA_LISTEN_PORT} -fallbackfee=0.00001 \
```

Comment:
On Bitcoin core v28.0.0 for some reason this wasn't using the port anymore. Using bind seems to work...

Reply:
Just tried bitcoin core v22-1 and it accepts `-bind=12345` but does not make a listening tcpip port. 

```sh
2024-12-25T07:55:25Z Command-line arg: bind="12345"
2024-12-25T07:55:25Z Command-line arg: server="1"
```
netstat -tl - no new port

`-bind=12345` _does_ make port:

```sh
tcp        0      0 0.0.0.0:12345           0.0.0.0:*               LISTEN     
tcp6       0      0 [::]:12345              [::]:*                  LISTEN     
```